src/searchPage/searchPageEvents.ts

```diff
@@ -509,3 +536,20 @@ export interface GeneratedAnswerCitationMeta {
     permanentId: string;
     id: string;
 }
+
+export type GeneratedAnswerFeedbackReason = 'irrelevant' | 'notAccurate' | 'outOfDate' | 'harmful' | 'other';
+
+export type GeneratedAnswerRephraseStrategy = 'stepByStep' | 'bulletPoints' | 'summarize';
```

Comment:
```suggestion
export type GeneratedAnswerRephraseFormat = 'stepByStep' | 'bulletPoints' | 'summarize';
```
I believe "format" is more used than "strategy"

Reply:
`GeneratedAnswerFormat` to match the updated design doc